Refit/MultipartItem.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refit
+{
+    public abstract class MultipartItem
+    {
+        public MultipartItem()
+        {
+
+        }
+
+        public MultipartItem(string fileName, string contentType)
+        {
+            this.FileName = fileName;
+            this.ContentType = contentType;
+        }
+
+        public virtual string FileName { get; set; }
+        public virtual string ContentType { get; set; }
+
+        public virtual HttpContent ToContent()
```

Comment:
I might've been a little too "open" with my API design. I can remove the virtuals and make public sets private